libraries/AP_GroundEffectControl/AP_GroundEffectController.h

```diff
@@ -0,0 +1,64 @@
+#pragma once
+
+#include <AP_AHRS/AP_AHRS.h>
+#include <PID/PID.h>
+#include <AP_RangeFinder/AP_RangeFinder.h>
+#include <Filter/ComplementaryFilter.h>
+
+class GroundEffectController {
+public:
+    GroundEffectController(AP_AHRS& ahrs, RangeFinder& rangefinder)
+        : _ahrs{ahrs}
+        , _rangefinder{rangefinder}
```

Comment:
I think its better to use the singletons for these rather than take refences.

Reply:
done